packages/marketplace/contracts/exchange/ExchangeCore.sol

```diff
@@ -58,12 +59,21 @@ abstract contract ExchangeCore is Initializable, TransferExecutor, ITransferMana
     );
     event OrderValidatorSet(IOrderValidator indexed contractAddress);
 
+    /// @notice event for setting new maximum for transfers
+    /// @param newMaxTransferValue new maximum
+    event MaxTransferSet(uint256 newMaxTransferValue);
+
     /// @notice initializer for ExchangeCore
     /// @param newOrderValidatorAddress new OrderValidator contract address
+    /// @param maxTransfer max number of transfers
     /// @dev initialize permissions for native token exchange
     // solhint-disable-next-line func-name-mixedcase
-    function __ExchangeCoreInitialize(IOrderValidator newOrderValidatorAddress) internal onlyInitializing {
+    function __ExchangeCoreInitialize(
+        IOrderValidator newOrderValidatorAddress,
+        uint256 maxTransfer
+    ) internal onlyInitializing {
         _setOrderValidatorContract(newOrderValidatorAddress);
+        transferMax = maxTransfer;
```

Comment:
Here is better to call:  _setMaxTransferValue so we emit

Reply:
alright, done